index.dd

```diff
@@ -175,7 +175,8 @@ void main()
 
 $(SECTION3 Power,
 
-$(P The best paradigm is to not impose something at the expense of others.
+$(P The ideal programming paradigm lets you choose the features you need today, 
+without sacrificing the ones you might need tomorrow.
```

Comment:
Heh, it's not that I like "you" so much, honest. :) But the first sentence is awkward -- it reads to me like, "the best viewpoint is the one where other people don't pay for your decisions." Even if we wrote, "The best [programming] paradigm imposes nothing at the expense of other things," it's still not true: all paradigms impose limits. I think the new wording is truer to what we are trying to express here.